vendor/github.com/moby/buildkit/vendor.conf

```diff
@@ -4,10 +4,10 @@ github.com/pkg/errors v0.8.0
 github.com/stretchr/testify v1.1.4
 github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
-golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
+golang.org/x/sys 1b2967e3c290b7c545b3db0deeda16e9be4f98a2
 
-github.com/containerd/containerd 08f7ee9828af1783dc98cc5cc1739e915697c667
-github.com/containerd/typeurl f6943554a7e7e88b3c14aad190bf05932da84788
+github.com/containerd/containerd a88b6319614de846458750ff882723479ca7b1a1
```

Comment:
Looks like docker/cli is behind; https://github.com/containerd/containerd/compare/08f7ee9828af1783dc98cc5cc1739e915697c667...a88b6319614de846458750ff882723479ca7b1a1

Reply:
updated with 051c4bb